docs/data-sources/ec_deployments.md

```diff
@@ -6,7 +6,7 @@ description: |-
 
 # Data Source: ec_deployments
 
-Use this data source to retrieve a list of deployment and resource kind IDs based on a supplied query.
+Use this data source to retrieve a list of IDs for the deployment and resource kind, based on the specified query.
```

Comment:
Each deployment will have IDs for different resource kinds, just realised this should have been plural from the beginning, sorry!

```suggestion
Use this data source to retrieve a list of IDs for the deployment and resource kinds, based on the specified query.
```